### PR TITLE
Remove `/learn` Redirections from V1-1 and Add Redirections to V1-2

### LINKED ISSUE #286: Remove `/learn` Redirections from V1-1 and Add Redirections to V1-2
**Description:**
With the removal of the /learn` version, we have done redirected its references to the v1-1 in [1].

We need to remove those and add redirections in the same to the new V1-2 release.

[1] https://github.com/ballerina-platform/ballerina-dev-website/pull/276

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
With the removal of the /learn` version, we have done redirected its references to the v1-1 in [1].

We need to remove those and add redirections in the same to the new V1-2 release.

[1] #276
> Fixes #286 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Comment by NipunaMarcus** [comment]
@praneesha can you remove these same redirections available in the v1-1

File: v1-1/learn.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ permalink: /v1-1/learn/
 redirect_from:
   - /v1-1/learn/tools-ides
   - /v1-1/learn/tools-ides/
-  - /learn/
```
Comment (NipunaMarcus) [inline v1-1/learn.md]: please remove the same redirections in each page in learn in v1-1 dir

Comment (praneesha) [reply v1-1/learn.md]: Done

**Comment by praneesha** [comment]
This is duplicated in https://github.com/ballerina-platform/ballerina-dev-website/pull/291.

**Pull request closed without merging**